Server/VikingVault.Services/AutomaticPaymentService.cs

```diff
@@ -54,5 +55,29 @@ public List<AutomaticPaymentDTO> GetAllAutomaticPayments(string token)
 
             return automaticPaymentData;
         }
+
+        public AutomaticPayment CreateAutomaticPayment(AutomaticPaymentDTO automaticPaymentDTO, string token)
+        {
+            var tokenObject = new JwtSecurityToken(token);
+            int userId = Int32.Parse(tokenObject.Payload["Id"].ToString());
```

Comment:
Use the same data type in the parse as it is in the variable declaration. (I know that `int` and `Int32` are the same but its confusing)